src/crypto/base58.mli

```diff
@@ -48,6 +48,8 @@ module Prefix : sig
   val ed25519_signature : string
   val secp256k1_signature : string
   val p256_signature : string
+
+  val chain_id : string
```

Comment:
Comment only about format. Since these types are grouped, is it possible to add one short comment line before each category?
Even if it is useless like:
```ocaml
  (* signatures definitions *)
  val ed25519_signature : string
  val secp256k1_signature : string
  val p256_signature : string
```
the goal is to add some guidelines for people coming from nowhere _(for example: me 🤣)_

Reply:
Hmm, this file is partially extracted from Tezos, as can be seen by the license on top of the file.

I would not agree that adding a comment would help, if someone doesn't notice the pattern at `_signature` they're even less likely to read the comment.

But please open an issue to discuss this,

Reply:
>Hmm, this file is partially extracted from Tezos

Ok, I didn't know that.

>But please open an issue to discuss this

Ok, I will open an issue when I will feel a need while coding to give concrete example.